### ISSUE TITLE
No explicit requirements on data format for MRI data

### ISSUE THREAD

**Issue by nbeliy** [bids-standard/bids-specification#1713]
Dear developers,

I can't find an explicit format requirements for MRI. 

All examples/templates implies Nifti format with one file (.nii). Will the Nifti 2 files (.hdr+.img) will also compatible with BIDS? Same question for DICOM images.

If BIDS restrict the file formats  for MRI (similar to [EEG restrictions](https://bids-specification.readthedocs.io/en/stable/modality-specific-files/electroencephalography.html#eeg-recording-data)), can these restrictions introduced into documentation?

P.S. Similar issue exists for PET, there is explicit mention of Nifti file, but without precision of one-file schema (.nii).

**Comment by sappelhoff** [comment]
It's under Common Principles:

> All imaging data MUST be stored using the NIfTI file format. 

https://bids-specification.readthedocs.io/en/latest/common-principles.html#imaging-files

where exactly would you have expected this information to be instead?

> Will the Nifti 2 files (.hdr+.img) will also compatible with BIDS

I don't think so

**Comment by Remi-Gau** [comment]
For DICOM I would redirect people towards this discussion / draft proposal

**Comment by nbeliy** [comment]
> It's under Common Principles:

Indeed, didn't fought to look there. But it will be better to place it into MRI explicitly. Other modalities provide the format in which data is expected.

Also, it leads to contradiction (saw it just now):

[Common Principles](https://bids-specification.readthedocs.io/en/latest/common-principles.html#imaging-files):
> All imaging data MUST be stored using the NIfTI file format. We RECOMMEND using compressed NIfTI files (.nii.gz), either version 1.0 or 2.0.

[Microscopy](https://bids-specification.readthedocs.io/en/latest/modality-specific-files/microscopy.html#file-formats):
> Microscopy raw data MUST be stored in one of the following formats:
[Portable Network Graphics](http://www.libpng.org/pub/png/) (.png)
[Tag Image File Format](https://en.wikipedia.org/wiki/TIFF) (.tif)
[OME-TIFF](https://docs.openmicroscopy.org/ome-model/6.1.2/ome-tiff/specification.html#) (.ome.tif for standard TIFF files or .ome.btf for [BigTIFF](https://www.awaresystems.be/imaging/tiff/bigtiff.html) files)
[OME-ZARR/NGFF](https://ngff.openmicroscopy.org/latest/) (.ome.zarr directories)

I guess MRI was written first, when imaging data meant MRI only.

**Comment by sappelhoff** [comment]
> I guess MRI was written first, when imaging data meant MRI only.

That is true. I think we should adjust this, thanks for bringing it up.

**Comment by Remi-Gau** [comment]
I agree that this section should better live under the datatype specific sections: namely MRI and PET (for now but with MRS soon).

I usually do not like duplication but in this case, I think most users will have the same expectation as @nbeliy on this one.

Plus we can macroify this if we want to avoid duplication.

**Comment by Remi-Gau** [comment]
@nbeliy do you feel like opening a PR for this one?

**Comment by nbeliy** [comment]
Will try, but will need an editor, I'm very dyslexic.

Just to be clear:

1. I would remove data format from Common principles (also check if there some other MRI specific stuff).
2. I would add Data format section into MRI
3. Add annex with data formats table for all defined modalities 

Is it good for you?

**Comment by Remi-Gau** [comment]
> Will try, but will need an editor, I'm very dyslexic.

No worries, codespell will do a first pass and we (maintainers) will help you along.

> Just to be clear:
> 
>     1. I would remove image data format from Common principles (also check if there some other MRI specific stuff).

yes

>     2. I would add Data format section into MRI

Given that Nifti are used for PET as well it will have to be added in the PET page too.

>     3. Add annex with data formats table for all defined modalities

I would say: let's start simple and keep that for another PR. Though I think some kind of summary table would indeed be welcome especially given the growing list of datatypes BIDS support.

Most likely this kind of table could actually be generated using the BIDS schema and some python code to create the table automatically. But as I said: let's keep that for another PR.

**Comment by nbeliy** [comment]
Started the PR (#1720)
